src/components/generators/view-components/string.vue

```diff
@@ -14,12 +14,41 @@
           {{ attribute.value }}
         </span>
       </template>
+      <template v-if="attribute.linkValue">
+        <span
+          id="copy-icon"
+          @click.prevent="copyOnClipboard(attribute.linkValue, $event)"
```

Comment:
$event is passed as parameter, but not used in the function?

And on a sidenote, the working term should be copyToClipboard to be more syntactically correct ;)